cli/cli/commands/cloud/load/load.go

```diff
@@ -0,0 +1,134 @@
+	"github.com/kurtosis-tech/kurtosis/cli/cli/kurtosis_config"
+	"github.com/kurtosis-tech/kurtosis/cli/cli/kurtosis_config/resolved_config"
+	"github.com/kurtosis-tech/kurtosis/contexts-config-store/store"
+	"github.com/kurtosis-tech/stacktrace"
+	"github.com/sirupsen/logrus"
+	"os"
+)
+
+const (
+	instanceIdentifierArgKey      = "instance-id"
+	instanceIdentifierArgIsGreedy = false
+	kurtosisCloudApiKeyEnvVarArg  = "KURTOSIS_CLOUD_API_KEY"
+)
+
+var LoadCmd = &lowlevel.LowlevelKurtosisCommand{
+	CommandStr:       command_str_consts.CloudLoadCmdStr,
+	ShortDescription: "Load a Kurtosis Cloud instance",
+	LongDescription: "Load a remote Kurtosis Cloud instance into the local context by providing the instance id." +
+		"Note, the remote instance must be in a running state for this operation to complete successfully",
+	Flags: []*flags.FlagConfig{},
+	Args: []*args.ArgConfig{
+		instance_id_arg.InstanceIdentifierArg(instanceIdentifierArgKey, instanceIdentifierArgIsGreedy),
+	},
+	PreValidationAndRunFunc:  nil,
+	RunFunc:                  run,
+	PostValidationAndRunFunc: nil,
+}
+
+func run(ctx context.Context, _ *flags.ParsedFlags, args *args.ParsedArgs) error {
+	instanceID, err := args.GetNonGreedyArg(instanceIdentifierArgKey)
+	if err != nil {
+		return stacktrace.Propagate(err, "Expected a value for instance id arg '%v' but none was found; "+
+			"this is a bug in the Kurtosis CLI!", instanceIdentifierArgKey)
+	}
+	logrus.Infof("Loading cloud instance %s", instanceID)
+
+	apiKey, err := loadApiKey()
+	if err != nil {
+		return stacktrace.Propagate(err, "Could not load an API Key. Check that it's defined using the "+
+			"%s env var and it's a valid (active) key", kurtosisCloudApiKeyEnvVarArg)
+	}
+
+	cloudConfig, err := getCloudConfig()
+	if err != nil {
+		return stacktrace.Propagate(err, "An error occured while loading the Cloud Config")
+	}
+	// Create the connection
+	connectionStr := fmt.Sprintf("%s:%d", cloudConfig.ApiUrl, cloudConfig.Port)
+	client, err := cloud.CreateCloudClient(connectionStr, cloudConfig.CertificateChain)
+	if err != nil {
+		return stacktrace.Propagate(err, "Error building client for Kurtosis Cloud")
+	}
+
+	getConfigArgs := &api.GetCloudInstanceConfigArgs{
+		ApiKey:     *apiKey,
+		InstanceId: instanceID,
+	}
+	result, err := client.GetCloudInstanceConfig(ctx, getConfigArgs)
+	if err != nil {
+		return stacktrace.Propagate(err, "An error occurred while calling the Kurtosis Cloud API")
+	}
+	decodedConfigBytes, err := base64.StdEncoding.DecodeString(result.ContextConfig)
+	if err != nil {
+		return stacktrace.Propagate(err, "Failed to base64 decode context config")
+	}
+
+	parsedContext, err := add.ParseContextData(decodedConfigBytes)
+	if err != nil {
+		return stacktrace.Propagate(err, "Unable to decode context config")
+	}
+
+	contextsConfigStore := store.GetContextsConfigStore()
+	// We first have to remove the context incase it's already loaded
```

Comment:
Really? Would throwing be better in that case? Erasing some data silently is a bit borderline IMO. We could add a `-f` option to force the override, but by default I'd just return with a nice message

Reply:
I don't fully agree that we are silently erasing data. We are replacing the config that was already there. Forcing it seems unnecessary in my opinion.